lib/rubocop/cop/internal_affairs/cop_description.rb

```diff
@@ -114,7 +114,7 @@ def comment_index(node, comment_line)
         end
 
         def relevant_file?(file)
-          file.match?(%r{/cop/.*\.rb\z})
+          file.match?(%r{/cop/.*\.rb\z}) && super
```

Comment:
After reverting the implementation by removing `super`, the newly added test case still pass. There might be some oversight in the test case.

```console
$ git diff .
diff --git a/lib/rubocop/cop/internal_affairs/cop_description.rb b/lib/rubocop/cop/internal_affairs/cop_description.rb
index d5097d4d3..769be81ac 100644
--- a/lib/rubocop/cop/internal_affairs/cop_description.rb
+++ b/lib/rubocop/cop/internal_affairs/cop_description.rb
@@ -114,7 +114,7 @@ module RuboCop
         end

         def relevant_file?(file)
-          file.match?(%r{/cop/.*\.rb\z}) && super
+          file.match?(%r{/cop/.*\.rb\z})
         end
       end
     end

$ bundle exec rspec spec/rubocop/cop/internal_affairs/cop_description_spec.rb

Randomized with seed 41144
........

Finished in 0.20845 seconds (files took 1.8 seconds to load)
8 examples, 0 failures

Randomized with seed 41144
```